config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :users
   root to: 'pages#home'
   # get '/profile', to: 'pages#profile', as: :profile
+  resources :profile
```

Comment:
Good idea to use resources! I would maybe exclude index and destroy explicitly?

`resources :profiles, except: [:index, :destroy]`

Just because if we allow a user to destroy their profiles then we might run into errors when a renter tries to click on the name of the provider - so the provider should always have a profile even if it only contains the bare minimum (first name, last name) 

I also think we might need plural - profileS - but maybe rails is clever enough to understand both!

Reply:
Ah! Good catch! Thanks! I'll fix that right now.